contrib/gitian/README.md

```diff
@@ -76,15 +80,16 @@ This setup is required to enable networking in the container.
 Docker
 ------
 
-Building in docker does not require much setup. Install docker on your host, then type the following:
+Prepare for building with docker:
 
 ```bash
-sudo apt-get install git make curl
+sudo apt-get install git make curl docker
```

Comment:
Did you check "docker" is the right package ? When I tried, it was dockerio or something, docker was something unrelated.

Reply:
Hmmm... I have both `docker` and `docker.io` installed. Let me check the dependencies, you are probably right.

Reply:
Nice catch, thanks!
Fixed now.